tiny_dnn/activations/activation_layer.h

```diff
@@ -101,6 +101,11 @@ class activation_layer : public layer {
                                    vec_t &dx,
                                    const vec_t &dy) = 0;
 
+  /**
+   * Target value range for learning.
+   */
+  virtual std::pair<float_t, float_t> scale() const = 0;
```

Comment:
> (Comment says it is target value range for learning, but it's all Greek to me...)

I would also like to understand this, just kept the docstring because it was _there_ before.